client/core/trade.go

```diff
@@ -1745,15 +1745,16 @@ func (t *trackedTrade) processRedemption(msgID uint64, redemption *msgjson.Redem
 	auth.RedemptionSig = redemption.Sig
 	auth.RedemptionStamp = redemption.Time
 
-	if dbMatch.Side == order.Maker {
-		if err := t.processTakersRedemption(match, redemption.CoinID); err != nil {
-			errs.addErr(err)
-		}
-	} else {
-		if err := t.processMakersRedemption(match, redemption.CoinID, redemption.Secret); err != nil {
+	if dbMatch.Side == order.Taker {
+		err = t.processMakersRedemption(match, redemption.CoinID, redemption.Secret)
+		if err != nil {
 			errs.addErr(err)
 		}
 	}
+	// The server does not send a redemption request to the maker for the
+	// taker's redeem since match negotiation is complete server-side when the
+	// taker redeems, and it is complete client-side when we redeem. If we are
+	// both sides, there is another trackedTrade and matchTracker for that side.
```

Comment:
Now client will not even respond if it gets a redemption request as maker.  We'll have to updated server pronto when this is merged.

Reply:
Correction, it still responds (`t.dc.ack` above), it just doesn't record the TakerRedeem coin anymore.  No server update urgency.